app/assets/stylesheets/common/_markus.scss

```diff
@@ -255,8 +246,8 @@ fieldset {
 
   // Fieldsets within fieldsets
   fieldset {
-    @include border-radius(0);
-    @include box-shadow(none);
+    border-radius: 0;

```

Comment:
Properties should be ordered border, border-radius, border-top, box-shadow, padding